storage/pruning/triePruningStorer.go

```diff
@@ -194,6 +197,22 @@ func (ps *triePruningStorer) GetFromLastEpoch(key []byte) ([]byte, error) {
 	return ps.activePersisters[lastEpochIndex].persister.Get(key)
 }
 
+// GetFromCurrentEpoch searches only the current epoch storer for the given key
+func (ps *triePruningStorer) GetFromCurrentEpoch(key []byte) ([]byte, error) {
+	ps.lock.RLock()
```

Comment:
I would keep the mutex only to get the storer pointer from the map.
Otherwise, all GetFromCurrentEpoch will be serialized.

Reply:
Done.